components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -208,9 +222,36 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
                 </p>
             );
         }
+
+        // if it is cloud and is in trial, in the case this component get's shown that means that the license hasn't been updated yet
+        // so let's notify to the user that the license
+        if (isCloud && isCloudTrial && subscription !== undefined) {
```

Comment:
Is it required to pass the entire subscription only to check if defined/undefined?
Maybe a boolean could be enough?